track/gbf.py

```diff
@@ -162,8 +162,8 @@ def get_order_confirmations(order_numbers):
     if "ShippingConfirmations" in confirmations:
         for shipping_confirmation in confirmations['ShippingConfirmations']:
             tracking_info[shipping_confirmation['OrderNumber']] = {
-                'date_kit_shipped': shipping_confirmation['ShipDate'],
-                'kit_tracking_n': shipping_confirmation['Tracking'],
+                'date_kit_shipped': shipping_confirmation['ShipDate'] if shipping_confirmation['ShipDate'] in shipping_confirmation else None,
+                'kit_tracking_n': shipping_confirmation['Tracking'] if shipping_confirmation['Tracking'] in shipping_confirmation else None,
                 #filter for items with return tracking numbers and returns tracking numbers
                 'return_tracking_n': [return_track for item in shipping_confirmation['Items'] if 'ReturnTracking' in item for return_track in item['ReturnTracking']],
```

Comment:
I'm wondering if we need to make sure that "Items" is in `shipping_confirmation` here.